tester/src/test_executors/auxboard.rs

```diff
@@ -142,19 +150,55 @@ impl TestExecutor for AuxBoardTestExecutor {
 			let mut l = self.logger.lock().unwrap();
 			l.in_progress("Handling messages...");
 		}
+
 		let start = chrono::Utc::now();
-		thread::sleep(time::Duration::from_millis(500));
+		let mut processed_messages = 0;
+		// Eat all and wait a bit
+		//self.bno.eat_all_messages(&mut self.delay);
+		//thread::sleep(time::Duration::from_millis(10));
+
+		let now = time::Instant::now();
+		while now.elapsed().as_millis() < 500
+		{
+			if self.int_pin.is_low()
+			{
+				processed_messages += self.bno.handle_one_message(&mut self.delay, 0);
+			}
+			thread::sleep(time::Duration::from_millis(1));
+		}
+		
+		if processed_messages > 0
```

Comment:
After enabling game rotation vector, we will get a response from the sensor. We can (and should) discard this response.

`eat_one_message` will eat exactly one message up to a timeout of 150ms, `eat_all_messages` will keep calling `eat_one_message` until nothing gets eaten, then return. I suspect that `eat_all_messages` will therefore hang, as the sensor will be sending quat data continuously. Lets just eat one or two, and see if that fixes it.

However, I suggest using `handle_one_message` instead because you can actually check if you got the number of messages you expected then.

Note: I intentionally do not use the INT pin here because I'm not sure that its actually necessary at this part of the test.
```suggestion
		// The total number of milliseconds until we treat this test failed.
		let ms_until_failure = 1000;
		// Experiment and see if it should be 1,2, or 3 messages. I suspect its just 1.
		let num_to_process = 2;
		let mut processed_messages = 0;
		let start = time::instant::now();
		while num_to_process > processed_messages && start.elapsed().as_millis() <= ms_until_failure {
			// Experiment with different timeouts. 150ms is what `eat_one_message()` does by default, but we can go much lower because of the loop - if we timeout, we will just retry!
			let timeout_ms = 5;
			processed_messages += self.bno.handle_one_message(&mut self.delay, timeout_ms);
		}
		
		if processed_messages >= num_to_process && start.elapsed().as_millis() <= ms_until_failure
```